src/plugins/table-of-contents/table-of-contents-display.js

```diff
@@ -18,21 +19,26 @@ export default function TableOfContentsDisplay({ content }) {
     setNodes(nodeList);
   }, [minLevel, maxLevel]);
 
-  if (!nodes.length) {
-    return null;
-  }
-
   return (
-    <ul className="TableOfContentsDisplay">
-      {nodes.map(node => (
-        <li
-          key={node.id}
-          className={`TableOfContentsDisplay-node TableOfContentsDisplay-node--indent${node.level - minLevel}`}
-          >
-          <a href={`#${node.id}`}>{node.text}</a>
-        </li>
-      ))}
-    </ul>
+    <div className="TableOfContentsDisplay">
+      {!!caption && (
+        <h1 className="TableOfContentsDisplay-caption">
+          <Markdown inline>{caption}</Markdown>
+        </h1>
```

Comment:
I have a feeling that if we format the title ourselves (which we don't do with other titles, but I do get that a TOC title makes no sense otherwise), I have a feeling we might need to offer a choice of headline size 🧐 But should we wait and see if that's necessary? What do you think?

Reply:
See my comment above.